mkosi.md

```diff
@@ -1126,6 +1126,12 @@ Those settings cannot be configured in the configuration files.
   rebuilding of these cached images, combine `-i` with `-ff` to ensure
   cached images are first removed and then re-created.
 
+`--debug=`
+
+: Enable additional debugging output. Takes a comma-separated list of
+  arguments specifying the area of interest. Pass any invalid value
+  (e.g. empty) to list currently accepted values.
```

Comment:
enclose `empty` in backticks?

Reply:
It was intended to mean really empty, i.e. `--debug=`. I want to rework it to allow `--debug=help` later on, so the exact phrasing here will change here anyway.